release/README.md

```diff
@@ -59,9 +74,13 @@ This process is going to trigger multiple CI checks as well as stage the release
 ## Add notes to release
 
 Once the PR has been fully merged, you are ready to create a new release for the newly created [tag](https://github.com/GoogleCloudPlatform/bank-of-anthos/tags).
+- Click the breadcrumbs on the row of the latest tag that was created in the [tags](https://github.com/GoogleCloudPlatform/bank-of-anthos/tags) page
+- Select the `Create release` option
 
 The release notes should contain a brief description of the changes since the previous release (like bug fixed and new features). For inspiration, you can look at the list of [releases](https://github.com/GoogleCloudPlatform/bank-of-anthos/releases).
 
+> ***Note:*** No assets needs to be uploaded. They are picked up automatically from the tagged revision
```

Comment:
s/needs/need

Reply:
Fixed in 7bb9544363ac55987a066e55c24a0fff2b5ab3b6